K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -193,32 +195,28 @@ public static void AddAggregates(this AggregateDictionary aggregateDictionary, s
                 }
 
                 // Column Metadata (Separator %)
-                // Structure: key%metric%value1%value2%keyed
                 var columnMetadata = column.ColumnName.Split(AggregationsConstants.MetadataSeparator);
+                string key;
 
-                if (columnMetadata.Length > 1)
+                if (columnMetadata.Length <= 1)
```

Comment:
can you add a comment before the if, explaing this condition and maybe return at the end of the scope and remove the nesting of the else part

Reply:
Added comment inside if/else. I prefer personally to keep the else, its more readable if someone check the code.

Reply:
@AsafMah is aligned with you. So you won my vote, I updated the code.